### DISCUSSION TITLE
Should we also use radicle?

### DISCUSSION THREAD

**Discussion by labra** [rudof-project/rudof#111] [General]
@jonassmedegaard [suggested in this issue](https://github.com/weso/shapes-rs/issues/77#issuecomment-2283845842) the use of [radicle](https://radicle.xyz/guides/user) to integrate contributions from people who don't want to use github. 

In principle, it seems very nice, although it also seems quite new. I am not sure if we should directly abandon github to embark on radicle or if we should try to use both at the same time, which I think is what Jones is suggesting.

As Jonas says in the issue, using both at the same time would require to use:

```
git push && git push rad
```

instead of a simple `git push`, and 

```
git pull rad && git push
``` 

when we want to integrate changes from `rad`.

According to [this discussion](https://radicle.zulipchat.com/#narrow/stream/369274-general/topic/GitHub.20.28GitLab.2C.20etc.29.20Bridge.3F), it seems that it is not possible to synchronize the issues in radicle with the issues in github, so I think, radicle would be used just as another git remote.

**Comment by jonassmedegaard** [comment]
> As I understand, it would require me to run:
>
> ```
> rad init
> ```
>
> in the root of the project and after that, try to keep the remote rad up-to-date, right?

Yes, that's the gist of it

> I think you are only suggesting to keep rad as another remote, not to replace github by rad, is that also right? At this moment, I would > prefer that option, I mean, to keep github and to add support for radicle so other contributors who don't want to use github, can also > contribute...

Yes, I suggest Radicle only as an optional addon git exchange, not dropping Github nor demoting Github nor syncronizing issues.

Obviously, if in some bright future Radicle becomes wildly popular, that position may change ;-)

**Comment by jonassmedegaard** [comment]
> As Jonas says in the issue, using both at the same time would require to use:
>
>```
>git push && git push rad
>```

I think that "require" is misleading: If only *some* developers have an interest in trying out Radicle, then they can *occationally* do `git push rad`, and those finding Radicle irrelevant can totally ignore it.

That leaves two questions, as I see it:
1. Do anyone with write access to this Github repo want to toy with Radicle?
2. Do anyone at all see a problem in others toying with Radicle - e.g. a security risk

Sounds like you, @labra, wants to play with Radicle which is adequate to answer 1).

Do anyone at all see a problem in others toying with Radicle - e.g. a security risk?
